pkg/api/schema/schema.proto

```diff
@@ -609,11 +609,14 @@ message ServerInfoResponse {
   // Unix timestamp (seconds) indicating when the server process has been started.
   int64 startedAt = 2;
 
+  // Total number of transactions across all databases.
+  int64 numTransactions = 3;
+
   // Total number of databases present.
-  int32 numDatabases = 3;
+  int32 numDatabases = 4;
```

Comment:
this is potential backward compability break
renumbering field will break existing clients or sdks, because these were working still on old proto

Reply:
The field has been introduced in this PR, I changed ordering in the context of different commits of this MR, so It should not represent a problem

Reply:
sorry, in review window I had only a change with renumbering index, not overall

Probably I had a view of changes after my last review